### ISSUE TITLE
[DOCS] Update remote cluster examples

### ISSUE THREAD

**Issue by lockewritesdocs** [elastic/elasticsearch#78216]
#77043 overhauled the remote cluster docs, and with it parts of CCR and CCS docs. @Erni indicated that our examples are using localhost (`127.0.0.1`) as `seeds`, which isn't real-world applicable. We should update all examples to include a separate remote cluster, and refer to that cluster alias and a seed node in the remote cluster.

**Comment by shainaraskas** [comment]
still needs to be done as of oct 2 2025, eg on https://www.elastic.co/docs/deploy-manage/remote-clusters/remote-clusters-api-key

@eedugon do you view this as a problem?

**Comment by eedugon** [comment]
good catch @shainaraskas , and now it's the perfect moment to improve this, as it will be low effort.
Do you want me to take care of it?

I agree providing examples not only valid for dev/test environments would be beneficial.

**Comment by shainaraskas** [comment]
@eedugon you can look at it when you have time or someone else can pick it up. just wanted to make sure that we should keep it open. I put it in the backlog
